treeherder/webapp/api/performance_data.py

```diff
@@ -979,10 +977,10 @@ def _create_graph_links(
         time_range_key = 'timerange'
         series_key = 'series'
 
-        highlighted_revisions_params = [
-            (highlighted_revision_key, new_revision[:12]),
-            (highlighted_revision_key, base_revision[:12]),
-        ]
+        highlighted_revisions_params = []
+        if base_revision:
```

Comment:
If the base revision is not provided then the graph link does not need to contain the short hash for it.